build-scripts/ose_images/ose_images.sh

```diff
@@ -426,31 +428,32 @@ check_builds() {
 
     if [ "$state" == "open" -o "$state" == "free" ]; then # free state means brew is waiting for a free builder
         echo "brew build for $package is still running..."
-    elif [ "$state" == "closed" ]; then
-        echo "==== ${package} IMAGE COMPLETED ===="
```

Comment:
Yeah... even I second guessed myself when I saw the diff. I took the elif/else parts and indented one level so they were their own if/else with a single call to `mv` at the end. I can confirm it's not just a whitespace change.